docs/source/ci.rst

```diff
@@ -37,17 +37,22 @@ remove the line that does ``flake8`` style-checking.
 Activate Travis-CI for Your GitHub Repository
 ---------------------------------------------
 
-.. warning::
-
-   The following instructions are not yet well tested.
-   `Feedback is welcome!  <https://github.com/NSLS-II/packaging-scientific-python/issues/new>`_
-
-1. Go to https://travis-ci.org and sign in with your GitHub account.
-2. Go to https://travis-ci.org/profile, which shows a list of your GitHub
-   repositories. Click the "Sync Account" button to refresh that list.
-3. Find your Python package in the list, and click the on/off switch to
-   activate Travis-CI on that repository.
-
-The next time you open a pull request or push a new commit to the master
-branch, Travis-CI will kick off a *build*, which you can see at
-``https://travis-ci.org/YOUR_GITHUB_USERNAME/YOUR_REPO_NAME/builds``.
+#. Go to https://travis-ci.org and sign in with your GitHub account.
+#. You will be prompted to authorize Travis-CI to access your GitHub account.
+   Authorize it.
+#. You will be redirected to https://travis-ci.org/profile, which shows a list
+   of your GitHub repositories. If necessary, click the "Sync Account" button
+   to refresh that list.
+#. Find your Python package in the list. Click the on/off switch next to its
+   name activate Travis-CI on that repository.
+#. Click the name, which will direct you to the list of *builds*
+   ``https://travis-ci.org/YOUR_GITHUB_USERNAME/YOUR_REPO_NAME/builds``. The
+   list will currently be empty. You'll see construction cones.
+#. The next time you open a pull request or push a new commit to the master
+   branch, Travis-CI will kick off a new build, and that list will update. As a
+   quick check, you can push an empty commit to master:
+
+   .. code-block:: bash
+
+      git commit --allow-empty -m "CI: Test Travis-CI integration."
+      git push origin master
```

Comment:
I'm curious if people think this is too dirty. This is what _I_ did to quickly check things are working, but I'm bit worried that it sets a bad example for people less versed in good practices. It is generally not good to throw empty commits at `master` for debugging purposes.... Should I remove this?

Reply:
eh, 50/50 on this.  I would just open a PR to test...

Reply:
K. Force-pushed an update to remove this.